src/doc/rustdoc/src/unstable-features.md

```diff
@@ -38,6 +38,20 @@ future.
 Attempting to use these error numbers on stable will result in the code sample being interpreted as
 plain text.
 
+### Custom CSS classes for code blocks
+
+```rust
+#![feature(custom_code_classes_in_docs)]
+
+/// ```class:language-c
+/// int main(void) { return 0; }
+/// ```
+fn main() {}
+```
+
+The text `int main(void) { return 0; }` is copied without formatting in a code block with the class
```

Comment:
What does "without formatting" mean?

Reply:
I changed for `without highlighting`, is it clearer ?

Reply:
Yes, that's better! I think "The text is rendered without highlighting" (not "copied") sounds better, but that's not a big deal.

Reply:
I changed for `rendered` while fixing the conflicts, thanks for the catch !